<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCredentials</class>
 <widget class="QDialog" name="DialogCredentials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credentials</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="labelClientSecret">
     <property name="text">
      <string>Client Secret</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEditClientSecret">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Client Secret</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushButtonConfirm">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEditRedirectUri">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Redirect URI</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEditClientId">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Client ID</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Input Credentials</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelRedirectUri">
     <property name="text">
      <string>Redirect Uri</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="labelClientId">
     <property name="text">
      <string>Client Id</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditClientId</tabstop>
  <tabstop>lineEditClientSecret</tabstop>
  <tabstop>lineEditRedirectUri</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonConfirm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
